### ISSUE TITLE
better error handling for invalid queries

### ISSUE THREAD

**Issue by cdignam-segment** [sbdchd/squawk#239]
If you run squawk against a file containing in invalid syntax, you get an unhelpful error message:

```
❯ squawk migrations/*                                    
Problem linting SQL files: Postgres failed to parse query
```

We should show a better error message:

```
1_initial_schema.sql: error: failed to lint file: postgres failed to parse query

2_add_emails.sql:2:1: warning: prefer-text-field

   2 | --
   3 | -- Create model Bar
   4 | --
   5 | CREATE TABLE "core_bar" (
   6 |     "id" serial NOT NULL PRIMARY KEY, 
   7 |     "alpha" varchar(100) NOT NULL
   8 | );

  note: Changing the size of a varchar field requires an ACCESS EXCLUSIVE lock.
  help: Use a text field with a check constraint.
```
